huaweicloud/services/iotda/resource_huaweicloud_iotda_dataforwarding_rule.go

```diff
@@ -327,7 +327,8 @@ func ResourceDataForwardingRuleCreate(ctx context.Context, d *schema.ResourceDat
 		return diag.FromErr(err)
 	}
 	for _, v := range targets {
```

Comment:
it's better to write like this:

```suggestion
	for i := range targets {
```

Reply:
mmodification completed

Reply:
modification completed